django-verdant/rca/static/rca/css/components/homepage-block.less

```diff
@@ -1,3 +1,53 @@
 .homepage-block {
     margin-bottom: @spacing-onepointfive;
+
+    &--full-width {
+        width: 100vw;
+        margin-left: -15px;
+
+        @media @media-desktop-small {
+            margin-left: -5vw;
```

Comment:
add some comments to explain that this is to make the content sit flush to the edge and become wider than the `page-content` parent div.